src/PresentationalComponents/RulesTable/Columns.js

```diff
@@ -32,7 +32,7 @@ export const Severity = {
 };
 
 export const Passed = {
-  title: 'Passed',
+  title: 'Rule state',
   sortByProp: 'compliant',
   renderExport: (rule) => (rule?.compliant ? 'Yes' : 'No'),
   renderFunc: renderComponent(PassedCell),
```

Comment:
Could we add `fitContent` from @patternfly/react-table? This will help the header not to wrap its text.

```suggestion
  renderFunc: renderComponent(PassedCell),
  transforms: [fitContent],
```

![image](https://user-images.githubusercontent.com/31385370/214298472-616f5dac-5959-4b16-b0d3-aa06b05f6f26.png)

Reply:
Agree, ack'ed suggesting. Thanks